Lib/KeychainAccess/Keychain.swift

```diff
@@ -535,8 +535,8 @@ public class Keychain {
         return self.dynamicType.prettify(itemClass: itemClass, items: items())
     }
     
-    #if os(iOS)
     @availability(iOS, introduced=8.0)
+    @availability(OSX, unavailable)
```

Comment:
Those build errors seems ignoring `#if os(iOS) #endif` directive.
So use `@availability(OSX, unavailable)` instead of that.